appsec-kit-starter/src/main/java/com/vaadin/appsec/service/AppSecServiceInitListener.java

```diff
@@ -37,7 +37,12 @@ public void serviceInit(ServiceInitEvent event) {
             appSecService.init();
             LOGGER.info("AppSec Kit initialized");
             appSecService.scanForVulnerabilities()
-                    .thenRun(appSecService::scheduleAutomaticScan);
+                    .exceptionally(appSecException -> {
+                        LOGGER.error(
+                                "Error scanning vulnerabilities. Automatic scan was not initiated.",
```

Comment:
Scheduling of the automatic scan happens only after this scan for vulnerabilities. I think the `Automatic scan was not initiated.` message is not needed here. If the scheduling of the automatic scan fails it is logged [there](https://github.com/vaadin/appsec-kit/pull/133/files#diff-af76497ec7eb3b5edebf5b7fc6b9c6014a7b83c4b9e9d2b4605ced2496413a03R225).

Reply:
I believe we can just call `scheduleAutomaticScan()` since it has an initial delay of `0` if no scan has been done previously or the minimum time between scans has passed. That way we can remove the entire `exceptionally` block.

If we are keeping the first redundant scan for vulnerabilities, then I will just remove the last sentence of the log message.

Reply:
The vulnerability scan after each startup is needed because during application development dependencies can change. If the user adds or removes a dependency then after application startup we want to provide up-to-date information about vulnerabilities immediately and to not wait until next automatic scan.

Reply:
Ok. Then I am just removing the last sentence.